ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__amd64-nightly.yaml

```diff
@@ -725,6 +725,19 @@ tests:
     - ref: idp-htpasswd
     - chain: openshift-e2e-test-qe-disasterrecovery-sanity
     workflow: cucushift-installer-rehearse-vsphere-ipi-proxy
+- as: vsphere-ipi-ovn-dualstack-privmaryv6-f4-disasterrecovery
+  cron: 7 1 * * 2,6
+  steps:
+    cluster_profile: vsphere-dis-2
```

Comment:
Why use `vsphere-dis-2` here?

Reply:
I follow vsphere bastion support job: vsphere-ipi-proxy-f14-disasterrecovery, it used vsphere-dis-2 and I'm not sure whether vsphere-2 is better option than vsphere-dis-2

Reply:
They are different purpose, for proxy job, we placed the cluster in a disconnected network segment, so we were using `vsphere-dis-2`, but for this one,  it was not. And `vsphere-dis-2` only has a small number of quota, we need to use it on specific purpose.

Reply:
ok, updated.